developing/README.md

```diff
@@ -177,7 +177,7 @@ docker-compose up -d
 Now, you should be ready to run your dev instance. I like to use `tmux` for this to run proxstar and the `rq worker` in separate panes.
 
 ```
-  flask run -p 8000 -h 0.0.0.0
```

Comment:
Hmm, do most people need to run with ssl?

Reply:
It's necessary for some routes such as `DELETE`. Also, it's for development. Is there a problem?

Reply:
👀 that's... wack